src/hooks/useAxiosWithAuth0.js

```diff
@@ -20,15 +24,19 @@ export default function useAxiosWithAuth0() {
   };
 
   useEffect(() => {
-    (async () => {
-      const token = await getAccessTokenSilently({
-        audience: AUTH0_AUDIENCE,
-      });
-      axiosWithAuth(token);
-    })();
-  }, []);
+    if (isAuthenticated) {
```

Comment:
This keeps the useEffect from running when the user is not logged in, which prevents previous error